src/test/kotlin/io/csbroker/apiserver/controller/v1/admin/AdminControllerIntegrationTest.kt

```diff
@@ -0,0 +1,109 @@
+import java.util.UUID
+
+class AdminControllerIntegrationTest : IntegrationTest() {
+    @Test
+    fun `어드민 유저 목록 조회`() {
+        // given + when
+        val response = request(
+            method = HttpMethod.GET,
+            url = "/api/admin/users/admin",
+            isAdmin = true,
+        )
+
+        // then
+        response.andExpect(status().isOk)
+            .andExpect {
+                val id = JsonPath.read(it.response.contentAsString, "$.data[0].id") as String
+                val user = findOne<User>("SELECT u From User u where u.id = :id", mapOf("id" to UUID.fromString(id)))
+                user.role shouldBe Role.ROLE_ADMIN
+            }
+    }
+
+    @Test
+    fun `단일 알림 생성`() {
+        // given
+        val user = save(
+            User(
+                email = "test-noti@test.com",
+                username = "test-noti",
+                password = "test-noti",
+                providerType = ProviderType.LOCAL,
+            ),
+        )
+
+        // when
+        val response = request(
+            method = HttpMethod.POST,
+            url = "/api/admin/notification",
+            body = NotificationRequestDto(
+                content = "test",
+                userId = user.id!!,
+                link = "https://test.com",
+            ),
+            isAdmin = true,
+        )
+
+        // then
+        response.andExpect(status().isOk)
+            .andExpect {
+                val id = JsonPath.read(it.response.contentAsString, "$.data.id") as Int
+                val notification = findOne<Notification>(
+                    "SELECT n FROM Notification n WHERE n.id = :id",
+                    mapOf("id" to id.toLong()),
+                )
+                notification.content shouldBe "test"
+                notification.link shouldBe "https://test.com"
+                notification.user.id shouldBe user.id
+            }
+    }
+
+    @Test
+    fun `알림 여러개 생성`() {
+        // given
+        val user = save(
+            User(
+                email = "test-noti2@test.com",
+                username = "test-noti2",
+                password = "test-noti2",
+                providerType = ProviderType.LOCAL,
+            ),
+        )
+
+        // when
+        val response = request(
+            method = HttpMethod.POST,
+            url = "/api/admin/notifications",
+            body = NotificationBulkInsertDto(
+                content = listOf(
+                    NotificationRequestDto(
+                        content = "test",
+                        userId = user.id!!,
+                        link = "https://test.com",
+                    ),
+                ),
+            ),
+            isAdmin = true,
+        )
+
+        // then
+        response.andExpect(status().isOk)
+            .andExpect {
+                val size = JsonPath.read(it.response.contentAsString, "$.data.size") as Int
+                size shouldBe 1
```

Comment:
```suggestion
                val size = JsonPath.read(it.response.contentAsString, "$.data.size") as Int
                size shouldBe 2
                val notifications = findAll<Notification>("SELECT n from Notification n")
                notifications.size shouldBe 2
```
테스팅을 할 때 Response 값을 확인하는 것도 중요하지만 실제로 DB에 잘 적재되었는지 확인하는 것 도 중요한 것 같아요